database/migrations/2020_03_25_055551_add_address_book.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddAddressBook extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('addressbooks', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->unsignedInteger('account_id');
+            $table->unsignedInteger('user_id');
+
+            $table->string('name', 500)->nullable();
+            $table->string('addressBookId', 100);
+            $table->timestamps();
+
+            $table->index('addressBookId');
+            $table->foreign('account_id')->references('id')->on('accounts')->onDelete('cascade');
+            $table->foreign('user_id')->references('id')->on('users')->onDelete('cascade');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
```

Comment:
We don't need down() method at all. We never rollback. I don't include them in my code ever.

Reply:
Well, I do.